core/embed/rust/src/ui/shape/utils/imagebuf.rs

```diff
@@ -0,0 +1,83 @@
+use crate::ui::{
+    geometry::Offset,
+    shape::{Bitmap, BitmapView, Canvas, CanvasBuilder},
+};
+
+/// Size of image buffer in bytes
+const IMAGE_BUFFER_SIZE: usize = 64 * 1024;
```

Comment:
We should make sure never to use this on F4 or MCUs with less RAM in general. Lets add conditional compilation for model_mercury for now (perhaps for this entire file), but we should figure out better way how to allocate proper amount of RAM for this (and perhaps graphics in general) according to UI requirements and MCU used